core/src/main/java/com/novoda/noplayer/drm/DrmSessionCreatorFactory.java

```diff
@@ -18,7 +18,7 @@ public DrmSessionCreator createFor(DrmType drmType, DrmHandler drmHandler) {
                 );
                 return StreamingDrmSessionCreator.newInstance(mediaDrmCallback, new FrameworkMediaDrmCreator());
             case WIDEVINE_MODULAR_DOWNLOAD:
-                return new NoDrmSessionCreator();
+                return new DownloadDrmSessionCreator((DownloadedModularDrm) drmHandler, new FrameworkMediaDrmCreator());
```

Comment:
it's for sugar 

`playerFactory.create(drmType, drmHandler)`

rather than 
```
playerFactory.createModularStreamingPlayer(modularStreamingHandler)
playerFactory.createModularDownloadPlayer(modularDownloadHandler)
playerFactory.createClassicPlayer(classicType)
playerFactory.createNoDrmPlayer(noDrmType)
```